llvm/include/llvm/ADT/DenseMap.h

```diff
@@ -905,7 +901,7 @@ class SmallDenseMap
 
   template <typename InputIt>
   SmallDenseMap(const InputIt &I, const InputIt &E) {
-    init(NextPowerOf2(std::distance(I, E)));
+    init(std::distance(I, E));
```

Comment:
BTW, this is potentially expensive for forward iterators... Nothing to do with this PR though